sale_commitment_date_mandatory/__manifest__.py

```diff
@@ -3,16 +3,17 @@
 
 {
     "name": "Sale commitment date mandatory",
-    "version": "14.0.1.0.1",
+    "version": "14.0.1.1.0",
```

Comment:
Please don't edit version numbers manually.

Reply:
When there is a related migration script I think it's better to edit this one manually. If a conflict appears, we would have to rename the migration script accordingly to match the version it fixes.